packages/web-demo/src/app/bootstrap/components/badge/badge-demo-page/badge-demo-page.component.ts

```diff
@@ -1,10 +1,14 @@
-import { Component, OnInit } from '@angular/core';
-const  codeTemplate = require('!!raw-loader!../badge-demo/badge-demo.component.html').default
+import { Component } from '@angular/core';
+const  badgeTemplate = require('!!raw-loader!../badge-demo/badge-demo.component.html').default;
+const  badgeNestedTemplate = require('!!raw-loader!../badge-nested-demo/badge-nested-demo.component.html').default;
+const  badgeInteractiveTemplate = require('!!raw-loader!../badge-interactive-demo/badge-interactive-demo.component.html').default;
+
 @Component({
   selector: 'app-badge-demo-page',
   templateUrl: './badge-demo-page.component.html'
 })
 export class BadgeDemoPageComponent  {
-  codeTemplate = codeTemplate;
-
+  badgeTemplate = badgeTemplate;
+  badgeNestedTemplate = badgeNestedTemplate;
+  badgeInteractiveTemplate = badgeInteractiveTemplate;
```

Comment:
Can you add the hide functionality for the demo app to make a more interactive example? The functionality does not need to be documented I think, implementation will vary wildly (e.g. if you display a list of applied filters badges will be rendered in a loop anyways).

Reply:
It has not been done for alerts/toast so we don't do it for badges neither.
And we will document this component with storybook anyway in the near future.